assets/js/base/hooks/use-previous.js

```diff
@@ -20,7 +20,7 @@ export const usePrevious = ( value, validation ) => {
 		) {
 			ref.current = value;
 		}
-	}, [ value, ref.current ] );
+	}, [ value, validation ] );
```

Comment:
Missing dep in a `useEffect()`. I needed to update calls to `usePrevious()` in other components to ensure `validation` doesn't change on each render (I commented it inline in those cases).